doc/rest-api.yaml

```diff
@@ -986,6 +986,11 @@ definitions:
           type: string
         type: array
         x-go-name: Profiles
+      project:
```

Comment:
This diff is missing the new `all-projects` parameter being added so this suggests that the swagger sections in `instances_get.go` need updating to add `all-projects`.

Reply:
I added swagger doc part before `instancesGet` function. Is this ok?